kafkajobs/config.go

```diff
@@ -30,6 +35,76 @@ func (c *config) InitDefault() ([]kgo.Opt, error) {
 	opts = append(opts, kgo.MaxVersions(kversion.Stable()))
 	opts = append(opts, kgo.RetryTimeout(time.Minute*5))
 
+	if c.TLS != nil && c.TLS.Enabled {
+		netDialer := &net.Dialer{Timeout: defaultSSLTimeout}
+		tlsDialerConfig := &tls.Config{MinVersion: tls.VersionTLS12}
```

Comment:
Would be also cool to create a similar function here: https://github.com/roadrunner-server/http/blob/master/tlsconf/default_tlsconf.go#L9.
We have to duplicate the code, because this is plugins, ideally, they should not depend on SDK or other plugins.

Reply:
Created func `tlsConfig`